packs/data/backgrounds.db/astrological-augur.json

```diff
@@ -0,0 +1,43 @@
+{
+    "_id": "5lcDlcXKD8eji6n3",
+    "img": "systems/pf2e/icons/default-icons/background.svg",
+    "name": "Astrological Augur",
+    "system": {
+        "boosts": {
+            "0": {
+                "value": [
+                    "cha",
+                    "wis"
+                ]
+            },
+            "1": {
+                "value": [
+                    "cha",
+                    "con",
+                    "dex",
+                    "int",
+                    "str",
+                    "wis"
+                ]
+            }
+        },
+        "description": {
+            "value": "<p>During your formative years, you learned the stars are a gateway to mystical power or glimpses into the future. Perhaps you grew up surrounded by Iruxi culture or were raised in one of the remaining Lirgeni sanctuaries. You've studied the constellations, using their alignment to predict fortune (or misfortune) and gaining magical powers unlike any of the other astrologers you know. Your readings might primarily focus on the Cosmic Caravan, or perhaps you use other stars to guide you, like the zodiac of Tian Xia.</p>\n<p>Choose two ability boosts. One must be to Wisdom or Charisma, and one is a free ability boost.</p>\n<p>You're trained in Astrology Lore. You can cast @Compendium[pf2e.spells-srd.Augury]{Augury} as a divine innate spell once per week, and the spell's effects can see up to 1 hour in the future rather than 30 minutes.</p>"
```

Comment:
```suggestion
            "value": "<p>During your formative years, you learned the stars are a gateway to mystical power or glimpses into the future. Perhaps you grew up surrounded by Iruxi culture or were raised in one of the remaining Lirgeni sanctuaries. You've studied the constellations, using their alignment to predict fortune (or misfortune) and gaining magical powers unlike any of the other astrologers you know. Your readings might primarily focus on the Cosmic Caravan, or perhaps you use other stars to guide you, like the zodiac of Tian Xia.</p>\n<p>Choose two ability boosts. One must be to Wisdom or Charisma, and one is a free ability boost.</p>\n<p>You're trained in Astrology Lore. You can cast <em>@Compendium[pf2e.spells-srd.Augury]{Augury}</em> as a divine innate spell once per week, and the spell's effects can see up to 1 hour in the future rather than 30 minutes.</p>"
```